_posts/2018/2018-04-03-what-a-code-of-conduct-means.md

```diff
@@ -0,0 +1,112 @@
+---
+title: 'What a code of conduct means'
+description: 'You should have one, but you should know how to act too.'
+layout: article
+category: other
+image:
+  base: '2018/what-a-code-of-conduct-means'
+  alt: ''
+  source: ''
+  credit: ''
+---
+
+I have a code of conduct for Brighton Ruby and have had for all five years of it’s existence.
+
+Last year was the first time I had to use it.
+
+I realise that there’s a lot of ‘why you should have one’ and not a lot of ‘what is it like to enforce one’. Enforcement is the _entire_ point of the code of conduct.
+
+I run the conference from my wholly-owned limited company. Thus legally Brighton Ruby is a private event. I have the ability just not to do ‘business’ with anyone I don’t wish to.
+
+I’m lucky in this respect as it allows me to run the event as a ‘benevolent dictator’ within whatever scope the nice people who generally attend will allow me.
+
+Anecdotally, I’ve heard of more reports of code of conduct breaches at a variety of events recently. I consider this to be a _good thing_. These things are happening anyway. It’s _much better_ to hear about and deal with them, than not to hear and leave those affected feeling unable to attend our events. It means we’re making our community better.
+
+This article lays out the process and principles that I currently use to maintain the safety of my attendees and encourage the best behaviour of everyone involved.
+
+# On fairness, intent and privilege
+
+I am a British, white, middle-class and (increasingly) middle-aged man. This means I cannot necessarily accurately understand the experiences of people who do not share my in-built societal privilege.
+
+I do my best as a ‘very visible for one day’ member of our community to empathise and ‘do right’ by individuals form groups who aren’t represented in our community in the same numbers as people who look like me.
+
+This means that I will trust individuals to accurately report their understanding of other’s behaviour as it effects them. And I will err on the side of trusting the opinion of those offended, rather than trying to judge solely from my perspective.
```

Comment:
I'm not a fan of the word "offended" here, as in it's common understanding doesn't quite cover the severity of what we're talking about in CoC violations IMO. Whatever word you choose here would need to encompass: threatened, made to fear for one's physical safety, assaulted, intimidated, humiliated, belittled, excluded, othered, etc, and I don't think "offended" quite covers those.

Reply:
Yeah. I’d removed it where I’d used it elsewhere.

Reply:
There may be another nuance here to be explicit about - that you take into account the power dynamics of the situation. In other words, would you respond the same to a white British middle aged man reporting that he feels unsafe because of something a young woman of color said, versus the opposite situation? “Believe the reporter” is an important principle, but must be tempered by an understanding of the power dynamics.